### DISCUSSION TITLE
Release Downloads?

### DISCUSSION THREAD

**Discussion by beeeeep54** [samolego/GolfIV#8] [General]
I am probably being very dumb, but I can't seem to find a place to download the built mod. If there is no place where I can download the built mod, could you perhaps add a download to the wiki or the readme or something? If you won't do that could you perhaps direct me to a tutorial on how to build a fabric mod, I tried, but I don't really understand the loom plugin thing. I would really appreciate it.

Sorry to bother you during the holidays.

**Comment by samolego** [comment]
Hi, since the mod is still in early beta, the stable downloads aren't available. (I'd like to redo the speed check before releasing 0.0.1 version :D).

You can download the builded file from [gh actions](https://github.com/samolego/GolfIV/actions). Click on the commit and download artifacts. It's a zip file, containing 4 files. Copy `golfiv-0.0.1.jar` to `mods` folder.

I'll also provide a guide on building the mod when I have time.

**Reply by beeeeep54** [reply]
Thanks so much!
